src/gui/accountsettings.cpp

```diff
@@ -222,6 +230,19 @@ void AccountSettings::createAccountToolbox()
     slotAccountAdded(_accountState);
 }
 
+
+void AccountSettings::slotNewMnemonicGenerated()
+{
+    ui->encryptionMessage->setText(tr("This account supports end-to-end encryption"));
+
+    QAction *mnemonic = new QAction(tr("Enable encryption..."), this);
```

Comment:
```suggestion
    QAction *mnemonic = new QAction(tr("Enable encryption"), this);
```

The ellipsis is not really needed here.

Reply:
It was requested by @camilasan to indicate that it opens up a new window

Reply:
Hm, I wouldn’t expect that from "…" a simple ellipsis. Also other buttons like "Add folder sync connection" and "Edit ignored files" which open a new window don’t do that. So I’d cut it for less noise.